packages/networks/src/networks/core.ts

```diff
@@ -0,0 +1,51 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const core: NetworkConfig = {
+  chain: 'core',
+  description:
+    'Build with EVM-compatible smart contracts on a Bitcoin-powered blockchain',
+  explorer: {
+    name: 'Core Explorer',
+    urls: {
+      address: (address) => `https://scan.coredao.org/address/${address}`,
+      base: 'https://scan.coredao.org/',
+      token: (address, holder) =>
+        `https://scan.coredao.org/token/${address}?a=${holder}`,
+      transaction: (hash) => `https://scan.coredao.org/tx/${hash}`,
+    },
+  },
+  featured: false,
+  id: 1116,
+  isTestNetwork: false,
+  multisig: '0x0eCbE42d1B8c9e56a9d358D3Ac1b5aaFAf5f07d5',
```

Comment:
I see that multisig only have a single signer. The best will be to create the safe using the available script `yarn hardhat safe:create` or if not possible add the signers specified here https://github.com/unlock-protocol/unlock/blob/2a1715bbcb325965c8b69ea9473303ae034081f7/governance/helpers/multisig.js#L22-L32

Reply:
Also there is a `yarn hardhat safe:check` to make sure everything is in order